lib/utils/couchstore/src/index.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+/*  eslint-disable max-len,no-param-reassign*/
```

Comment:
Can you please adhere to the defined chars limit across the Abacus sub-modules?
We intend to increase this 80 chars to 120 soon, but this is yet another reason to leave the max-len out of disable list, so we can validate everything is ok.